src/H5FDros3_s3comms.c

```diff
@@ -669,6 +681,14 @@ H5FD__s3comms_s3r_open(const char *url, const H5FD_ros3_fapl_t *fa, const char *
                     aws_error_str(aws_last_error()));
     cond_var_init = true;
 
+    /* Check if path-style requests should be forced */
+    pathstyle_env = getenv(HDF5_ROS3_VFD_FORCE_PATH_STYLE);
```

Comment:
This is needed for s3proxy testing, as I couldn't get virtual-hosted-style requests to work correctly with it, but path-style requests work just fine.